### PR TITLE
fix EOL crash, fix undefined resources crash

### PR DESCRIPTION
**Pull request by drudrum**
Ignore EOL errors, fix undefined resources crash

### CODE REVIEW

File: lib/object/object.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ class PDFObject {
 
     lexer.shift(3)
 
-    lexer.skipEOL(1)
+    lexer.skipEOL(1,true)
```
Comment (rkusa) [inline lib/object/object.js]:
```suggestion
    lexer.skipEOL(1, true)
```

File: lib/object/object.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ class PDFObject {
           throw new Error('Invalid stream: `endstream` not found')
         }
 
-        lexer.skipEOL(1)
+        lexer.skipEOL(1,true)
```
Comment (rkusa) [inline lib/object/object.js]:
```suggestion
        lexer.skipEOL(1, true)
```

**Comment by rkusa** [comment]
@drudrum Do you maybe also have an example PDF that can be shared publicly for which this change is necessary, so I could add a test for it? If not, do you maybe know with which program the failing PDF was created?

**Comment by drudrum** [comment]
I use it to combine russian AIP. https://github.com/drudrum/oneDocAipRus

**Comment by drudrum** [comment]
Later i can found some documents from AIP Russia.

**Comment by mmyjona** [comment]
I also encounter this bug. But in my case, the whole resources is null.
My pdf is generated with chrome.
I also summitted a [PR ](https://github.com/rkusa/pdfjs/pull/150)

**Comment by rkusa** [comment]
Cherry-picked into master here https://github.com/rkusa/pdfjs/commit/0a095e9edbc194e3c9faac969e41a81a26b2bd0a and here https://github.com/rkusa/pdfjs/commit/09f7f72c7f708681def6219b07595b56b689e4a4. Thanks again for the PR

**Pull request closed without merging**